packages/@tinacms/cli/src/cmds/audit/validations/validateMutations.ts

```diff
@@ -0,0 +1,122 @@
+import { AuditError, AuditIssue } from '../issue'
+import { assertShape } from '@tinacms/graphql'
+import { AuditArgs } from '../auditArgs'
+
+export const validateMutations = async (
```

Comment:
Mostly unchanged, but pulled this section in its own file to keep the higher level "audit" clean